style.css

```diff
@@ -0,0 +1,435 @@
+/* Importing Google fonts */
+@import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');
+
+* {
+  padding: 0%;
+  box-sizing: border-box;
+  --bg-blue: #2874f0; 
+  font-family: 'Roboto', sans-serif;
+  letter-spacing: 1;
+}
+/* Header */
+header {
+  height: 56px;
+  background-color: var(--bg-blue);
```

Comment:
Please explain this var() function in CSS

Reply:
It is used for accessing CSS variables. Instead of using same css property over and over again it is good to use CSS variables